pages/index.md

```diff
@@ -38,7 +38,7 @@ description: From personal blogs to massive documentation sites, Markdoc is a co
 
   {% item %}
 
-  #### Developer & writer friendly {% .jumbo %}
+  ### Developer & writer friendly {% .h4 .jumbo %}
```

Comment:
feels odd to me we are giving h3s a h4 class. Why do we need to do this?

Reply:
This is a super common CSS pattern. It basically means that semantically, we want this element to be an h3, but _styled_ as an h4.

Reply:
I can look at alternatives though 👍

Reply:
Ah, got it